client/blocks/get-apps/mobile-download-card.tsx

```diff
@@ -1,14 +1,12 @@
 import { useTranslate } from 'i18n-calypso';
 import JetpackAppLogo from 'calypso/assets/images/icons/jetpack-app-logo.svg';
 import QrCode from 'calypso/blocks/app-promo/qr-code';
-import userAgent from 'calypso/lib/user-agent';
+import { isIos, isMobile } from 'calypso/lib/user-agent';
 import AppsBadge from './apps-badge';
 import { AppsCard } from './apps-card';
 
 const MobileDownloadCardTest = () => {
```

Comment:
Not sure why this has the `*Test` suffix in name, let's rename it as `MobileDownloadCard`.

Reply:
> Not sure why this has the `*Test` suffix in name, let's rename it as `MobileDownloadCard`.

👍 Renamed as part of d6ff89832c0.